apps/src/makerlab/BoardController.js

```diff
@@ -195,8 +202,12 @@ function initializeCircuitPlaygroundComponents() {
       freq: 100
     }),
 
-    gyro: new five.Gyro({
+    accelerometer: new five.Gyro({
```

Comment:
Great question, someone noted the component itself is technically an accelerometer so wanted to be accurate. It actually looks like there's a `five.Accelerometer` I most likely will change the PlaygroundIO plugin to extend instead.

Re: usage, another good Q, this is not referenced in the codebase yet, though I updated a temporary usage instructions doc. The API is subject to change a good deal over the next couple of weeks.